test/ginkgo/resize_test.go

```diff
@@ -0,0 +1,113 @@
+package test_test
+
+import (
+	//"bytes"
+	//"encoding/json"
+	//"fmt"
+	//"io"
+	//"os"
+	//"os/exec"
+	//"os/user"
+	//"path/filepath"
+	"runtime"
+	//"strings"
+	//"syscall"
+	//"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	//"github.com/sirupsen/logrus"
+)
+
+var _ = Describe("start flags", func() {
+
+	Describe("memory, cpus, disk", func() {
+
+		It("setup CRC", func() {
+			Expect(RunCRCExpectSuccess("setup")).To(ContainSubstring("Setup is complete"))
+		})
+
+		It("start CRC with custom memory, cpu, and disk", func() {
+			Expect(RunCRCExpectSuccess("start", "-b", bundleLocation, "-p", pullSecretLocation, "--memory", "12000", "--cpus", "6", "--disk-size", "40")).To(ContainSubstring("Started the OpenShift cluster"))
+		})
+
+		switch os := runtime.GOOS; os {
+		case "linux":
+
+			It("check memory size", func() {
+				out, err := RunOnHost("ssh", "-i", credPath, "core@192.168.130.11", "cat", "/proc/meminfo")
```

Comment:
You can use the same code as collect.go:

```
	client := machine.NewClient(constants.DefaultName, true, network.DefaultMode)
	ip, err := client.IP()
	if err != nil {
		return err
	}
	ssh, err := ssh.NewClient(constants.DefaultSSHUser, ip, 22, &ssh.Auth{
		Keys: []string{constants.GetPrivateKeyPath()},
	})
	if err != nil {
		return err
	}
	ssh.Output("cat /proc/meminfo")
```

It will work for all platforms o/

Reply:
I rewrote it using this code. Thanks! However, it's the crc code that we're now using to test CRC. It raises questions :). I remember I once tried to use the sshRunner function but was told that it's better not to use CRC code (which, strictly speaking, makes sense). We can discuss on Slack.